### PR TITLE
Initial Rackspace Cloud Big Data how-to docs

### PR DESCRIPTION
**Pull request by PaulMontgomery**

### CODE REVIEW

**Comment by rs-randallburt** [comment]
Mostly LGTM:
- Might want to explain the trickier properties like public key stuff.
- Include info about the available stacks and flavor in the Resource Section (in addition to what you have in the reference but in a more conversational format)
- Explain what each output is and what its for.

I know a lot of this info is in the documentation for the resource itself, but it is nice to have it here as well.

File: cloud-bigdata.rst [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        description: Cloud Big Data Cluster ID
+
+      cbd_version:
+        value: { get_attr: [cbd_cluster, cbdVersion] }
+        description: Cloud Big Data version
+
+Reference Documentation
+=======================
+
+- `Rackspace Cloud Big Data Developer Guide, v1 <http://docs.rackspace.com/cbd/api/v1.0/cbd-devguide/cbd-devguide-20150423.pdf>`__
```
Comment (cryptickp) [inline cloud-bigdata.rst]: Can you add [v2](http://docs.rackspace.com/cbd/api/v1.0/cbd-devguide-2/cbd-devguide-2-20150630.pdf) guide instead.

**Comment by PaulMontgomery** [comment]
Thanks for the recommendations, updated.

**Comment by cryptickp** [comment]
:+1:

**Comment by rs-randallburt** [comment]
Awesome descriptions! You should update the property and output descriptions of the resource itself to match these since they're so much better!

:+1:

**Comment by rs-randallburt** [comment]
@rajanarun can you please give this a look-see and either :+1:  or :-1:  Thanks!

**Pull request merged**